BraintreeCore/schemas/com.braintreepayments.api.AnalyticsDatabase/6.json

```diff
@@ -0,0 +1,64 @@
+{
+  "formatVersion": 1,
+  "database": {
+    "version": 6,
+    "identityHash": "e602d210f4642f4843a9b29a026c81cb",
+    "entities": [
+      {
+        "tableName": "analytics_event",
+        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`name` TEXT NOT NULL, `timestamp` INTEGER NOT NULL, `_id` INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, `payPalContextId` TEXT, `linkType` TEXT, `isVaultRequest` INTEGER NOT NULL)",
```

Comment:
This approach doesn't work for us as it puts all the items in the db.
Consider converting the payload into a json string and dumping into a single column.